src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -50,6 +56,7 @@ public class Constants {
   public static final int MAX_STREAMING_INGEST_API_CHANNEL_RETRY = 3;
   public static final int STREAMING_INGEST_TELEMETRY_UPLOAD_INTERVAL_IN_SEC = 10;
   public static final long EP_NDV_UNKNOWN = -1L;
+  public static final int MAX_REFRESH_TOKEN_RETRY = 3;
```

Comment:
is this only used for OAuth? if yes, could you add it to the name to make it clear?

Reply:
Yes, modified.